src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,32 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        User user;
+        try {
+            user = authenticationService.register("bobemail@com", "bob_pass");
```

Comment:
The email 'bobemail@com' used in the `register` method might not be valid due to the missing top-level domain (TLD). Consider using a valid email format, such as 'bobemail@example.com'.